torch/onnx/symbolic_opset9.py

```diff
@@ -447,6 +448,9 @@ def prelu(g, self, weight):
 
 
 def relu(g, input):
+    if input in sym_help._quantized_ops:
```

Comment:
My suggestion is that, let's override the relu symbolic in symbolic_caffe2.py. It will make our change non-intrusive to the regular symbolics.

Reply:
When we process the symbolics, we process the aten::relu function and arrive here first. quantized relu does not have a `quantized` prefix in the name, so there is no way to know if it is quantized or not without checking the input.